src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +37,28 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={query}
+        onChange={(event) => {
```

Comment:
in such cases better to create a separate function instead of passing an anonymous function
for example: `handleInputChange`. the same for select and clear button